go.md

```diff
@@ -43,6 +43,8 @@ flowchart LR
   click chainlink-solana href "https://github.com/smartcontractkit/chainlink-solana"
   chainlink/v2 --> chainlink-starknet/relayer
   click chainlink-starknet/relayer href "https://github.com/smartcontractkit/chainlink-starknet"
+  chainlink/v2 --> grpc-proxy
```

Comment:
this file was making CI to fail after `github.com/smartcontractkit/grpc-proxy v0.0.0-20240830132753-a7e17fec5ab7` was added. 

`make generate` solved it by adding these lines